contrib/examples/actions/workflows/mistral-yaql-st2kv-user-scope.yaml

```diff
@@ -0,0 +1,12 @@
+version: '2.0'
+
+examples.mistral-yaql-st2kv-user-scope:
+    vars:
+        polo: unspecified
+    tasks:
+        task1:
+            action: std.noop
+            publish:
+                polo: <% st2kv('marco') %>
```

Comment:
This is slightly confusing for me. I think we should explicitly use st2kv.get(). There can be cases when we want to publish a variable to kv store. @m4dcoder tells me that YAQL really doesn't like `.` because it has a special meaning in YAQL world. I don't quite get that. I think st2kv_get is another option or if we can do st2kv['marco'], that is another option.